src/app/components/profile/profile.component.html

```diff
@@ -1,76 +1,126 @@
 <app-header-static></app-header-static>
-
-<div class="mid-container">
-	<div class="container-top profile-container">
-		<div class="align-center">
-			<img class="profile-photo" src="assets/images/profile.png" />
-		</div>
-		<div class="item">
-			<div class="content fw-regular">
-				{{user.username}}
-			</div>
-			<div class="fs-15">
-				{{user.email}}
-			</div>
-		</div>
-		<div class="user-details">
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					First Name<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.first_name}}
-				</div>
-			</div>
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					Last Name<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.last_name}}
-				</div>
-			</div>
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					Affiliated to<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.affiliation}}
-				</div>
-			</div>
-		</div>
-		<div class="item">
-			<div class="highlight">
-				<span class="fw-regular fs-14">Profile Completed</span>
-				<br> <span class="text-med-black">{{pcomp}}</span>
-				<div>
-				</div>
-				<br>
-				<div class="item">
-					<button class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark
-						fs-16 rm-pad-top" (click)="isTokenModalVisible= true;">Get your Auth Token</button>
-				</div>
-				<div class="item change-password">
-					<a (click)="updatePassword()" class="dark-link pointer">
-						<strong class="fw-semibold fs-15 change-password-text">Change Password</strong>
-					</a>
-					<button (click)="updateUserDetails()" class="btn btn-waves-effect
-						waves-dark grad-btn grad-btn-dark rm-pad-top fs-16">Update Profile</button>
-				</div>
-
-				<div class="modal-container" [class.hidden]="!isTokenModalVisible">
-					<div class="modal-card">
-						<div class="title">
-							API Authentication Token
+<div class="main_content">
+	<app-side-bar style="background-color: rgb(250,250,250)"></app-side-bar>
+	<div class="container-fluid main_content">
+		<div class="row">
+			<div class="dashboard">
+				<div class="md-container  main_content" id="card_container">
```

Comment:
Please remove the extra space after `md-container` class name.

Reply:
sir the hyphen is coming from the database because it is empty so how can i change it?

Reply:
No, it is not coming from database but we are modifying it in our typescript here (https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/app/components/profile/profile.component.ts#L104) in order to display hyphen in our UI. So you just check if it's hyphen then send empty string `""`.

Please let me know if you didn't get this.

Reply:
thx sir

Reply:
i cant seem to find how can i make submit button work

Reply:
and from where the comment of enter the first name is coming?

Reply:
Please take a look how `affiliation, firstname or lastname` fields here (https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/app/components/utility/modal/modal.component.ts#L244) are being checked if they are modified or not and accordingly `Submit` button gets enabled.

Reply:
Please let me know if you have more doubts.

Reply:
sir i still cant understand this